code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -1413,3 +1635,127 @@
 			new /obj/item/wisp_lantern(src)
 		if(3)
 			new /obj/item/prisoncube(src)
+
+/obj/item/twohanded/crucible
+	name = "Crucible Sword"
+	desc = "Made from pure argent energy, this sword can cut through flesh like butter."
+	icon = 'icons/obj/lavaland/1x2.dmi'
+	icon_state = "crucible0"
+	var/icon_state_on = "crucible1"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+	item_state = "crucible0"
+	force = 3
+	throwforce = 5
+	throw_speed = 3
+	throw_range = 5
+	force_wielded = 24
+	w_class = WEIGHT_CLASS_NORMAL
+	var/w_class_on = WEIGHT_CLASS_HUGE
+	hitsound = "swing_hit"
+	var/hitsound_on = 'sound/weapons/bladeslice.ogg'
+	armour_penetration = 50
+	light_color = "#ff0000"//BLOOD RED
+	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
+	block_power = 50
+	block_level = 1
+	block_upgrade_walk = 1
+	block_flags = BLOCKING_ACTIVE | BLOCKING_NASTY | BLOCKING_PROJECTILE
+	max_integrity = 400
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 100, "wound" = 35)
```

Comment:
why does this have wound armor? we dont have wounds, and objects cant get wounds